### DISCUSSION TITLE
bug: Body of chat and title API requests are being logged.

### DISCUSSION THREAD

**Discussion by theasp** [open-webui/open-webui#3207] [General]
# Bug Report

## Description

**Bug Summary:**

The body of chat API requests is now being logged.  This probably shouldn't happen by default.  The title endpoint is also affected.  See output below.

It looks like this was introduced here:  https://github.com/open-webui/open-webui/commit/88d053833d5287b924a419fb3f926b41c348a3b4

**Steps to Reproduce:**

1. Run Open WebUI in a terminal or in Docker
2. Start chatting with a model
3. Check output or docker logs

**Expected Behavior:**

It shouldn't log request bodies by default.

**Actual Behavior:**

Request bodies logged.

## Environment

- **Open WebUI Version:** 0.3.4
- **Ollama (if applicable):** N/A

- **Operating System:** Ubuntu 22.04
- **Browser (if applicable):** Chrome

## Reproduction Details

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [X] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
N/A

**Docker Container Logs:**
```
open-webui-1  | INFO:apps.ollama.main:url: http://ollama:11434
open-webui-1  | {'model': 'DEFAULT/llama3-8b:latest', 'messages': [{'role': 'user', 'content': "Do people ever say things to LLMs that shouldn't be logged?"}], 'options': {}}
open-webui-1  | INFO:     192.168.2.125:0 - "POST /ollama/api/chat HTTP/1.1" 200 OK
open-webui-1  | INFO:     192.168.2.125:0 - "POST /api/v1/chats/274b9d3d-1b5c-4434-a2ea-7bc30723f6a8 HTTP/1.1" 200 OK
open-webui-1  | INFO:     192.168.2.125:0 - "GET /api/v1/chats/ HTTP/1.1" 200 OK
open-webui-1  | generate_title
open-webui-1  | *llama3-8b
open-webui-1  | {'model': '*llama3-8b', 'messages': [{'role': 'user', 'content': "This query will be summarized:\nDo people ever say things to LLMs that shouldn't be logged?\n\nIgnore all pr
evious instructions.\n\nGenerate a concise and accurate title (no more than 5 words) that summarizes the query in natural English. The title can start with a suitable Emoji to enhance unders
tanding, but avoid quotation marks or special formatting. RESPOND ONLY WITH THE TITLE TEXT.\n\nExample titles:\n📉 Stock Market Trends\n🍪 Perfect Chocolate Chip Recipe\n🎵 Evolution of Musi
c Streaming\nRemote Work Productivity Tips\n🤖 Artificial Intelligence in Healthcare\n🎮 Video Game Development Insights"}], 'stream': False, 'max_tokens': 50, 'chat_id': '274b9d3d-1b5c-4434
-a2ea-7bc30723f6a8', 'title': True}
open-webui-1  | INFO:apps.ollama.main:url: http://ollama:11434
```

**Screenshots (if applicable):**
N/A

## Installation Method

Docker

## Additional Information

N/A

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by theasp** [comment]
I think these prints should be removed or converted into `log.debug()`:

```
backend/apps/openai/main.py:433:    print(payload)
backend/apps/openai/main.py:438:    print(payload)
backend/apps/ollama/main.py:843:    print(payload)
backend/main.py:776:    print(payload)
backend/main.py:840:    print(payload)
```

**Comment by theasp** [comment]
@tjbck I'm curious, when bug reports get converted to discussions, does that mean the bug report is invalid or it's not actually a bug?

**Reply by tjbck** [reply]
We'd like to keep only the issues that are reproducible by me or core issue that impact the overall usage of the webui to our issues section! Feel free to make a PR regardless!

**Reply by theasp** [reply]
Thanks for your reply!  PR for converting them to log.debug() here: https://github.com/open-webui/open-webui/pull/3217
